core/arch/arm/kernel/secure_partition.c

```diff
@@ -594,6 +670,12 @@ static TEE_Result sp_init_uuid(const TEE_UUID *uuid, const void * const fdt)
 	if (res)
 		goto out;
 
+#ifdef CFG_CORE_TPM_EVENT_LOG
```

Comment:
Please use `if (IS_ENABLED(CFG_CORE_TPM_EVENT_LOG))` instead

Reply:
Done, added empty handle_tpm_event_log implementation for CFG_CORE_TPM_EVENT_LOG=n.